src/app/host/host-page/host-page.component.html

```diff
@@ -1,7 +1,7 @@
 <app-header [text]="header"></app-header>
 <app-container>
-    <p *ngIf="averageRating != null">Average Rating: {{ averageRating }}</p>
+    <p *ngIf="host.averageRating" class="mat-headline-5" style="margin-left: 1em; margin-top: 1em">Average Rating: {{ host.averageRating }}</p>
     <div class="reviews">
-        <app-reviews [id]="id || 0" [reportingAllowed]="reportingAllowed" *ngIf="reviews != null" (reviewDeleted)="calculateAverageRating()"></app-reviews>
+        <app-reviews [id]="id || 0" [reportingAllowed]="reportingAllowed" *ngIf="reviews != null"></app-reviews>
```

Comment:
I suggest leaving this line as is.
```suggestion
        <app-reviews [id]="id || 0" [reportingAllowed]="reportingAllowed" *ngIf="reviews != null" (reviewDeleted)="calculateAverageRating()"></app-reviews>
```